packages/vector_graphics_compiler/lib/src/svg/opacity_peephole.dart

```diff
@@ -103,6 +103,24 @@ class _OpacityForwarder extends Visitor<Node, _ForwardResult>
   Node visitViewportNode(ViewportNode viewportNode, _ForwardResult data) {
     throw UnsupportedError('Cannot forward opacity through a viewport node');
   }
+
+  @override
+  Node visitResolvedVerticesNode(
+      ResolvedVerticesNode verticesNode, _ForwardResult data) {
+    // TODO: find a way to use tighter bounds from the vertices.
```

Comment:
Iterate them and accumulate their points into an expanding rect

Reply:
But I don't want an expanding rect - I can use the existing path bounds. What I want to use is the actual polygon intersection, since that would give much tighter bounds. That may not be worth implementing in Dart if we can use an existing library though

Reply:
ahh you want to know if vertices intersect?

Reply:
One thing we should be able to do with vertices is just tessellate with the right winding rules so that intersecting things disappear

Reply:
Well for the opacity optimization, i'd need tighter bounds - so that I can apply it more often, but I don't think its that important. I don't necessarily need to compute the intersections - though I guess we'd need it for clipping